utils/zerovec/src/yoke_impls.rs

```diff
@@ -60,25 +60,19 @@ unsafe impl<'a, T: 'static + AsVarULE> Yokeable<'a> for VarZeroVec<'static, T> {
 
 /// This impl can be made available by enabling the optional `yoke` feature of the `zerovec` crate
 #[allow(clippy::transmute_ptr_to_ptr)]
-unsafe impl<
-        'a,
-        K: 'static + ZeroMapKV<'static> + Yokeable<'a>,
-        V: 'static + ZeroMapKV<'static> + Yokeable<'a>,
-    > Yokeable<'a> for ZeroMap<'static, K, V>
-where
-    K::Output: ZeroMapKV<'a>,
-    V::Output: ZeroMapKV<'a>,
+unsafe impl<'a, K: 'static + for<'b> ZeroMapKV<'b>, V: 'static + for<'b> ZeroMapKV<'b>> Yokeable<'a>
+    for ZeroMap<'static, K, V>
```

Comment:
Nit (optional): I find it more readable when type bounds that are longer than one token are moved to `where`